SECURITY.md

```diff
@@ -0,0 +1,5 @@
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within Jest, please submit a report via the Github's Private Vulnerability Reporting feature.
```

Comment:
I would suggest linking "Github's Private Vulnerability Reporting" to https://github.com/jestjs/jest/security/advisories

Other than that this looks good to me!